src/styles/header.scss

```diff
@@ -0,0 +1,21 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: $white;
+  width: calc(100% - #{$header-padding * 2});
+  padding: 0 50px;
+
+  .logo {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: $logo-size;
+    width: $logo-size;
```

Comment:
The variable `$logo-size` is used here but not defined in this file. Ensure it is defined in an imported file or within this SCSS file.